examples/shopping/get_product_bidding_category_constant.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import absolute_import
+ 
+import argparse
+import six
+import sys
+import uuid
+import collections
+import google.ads.google_ads.client
+from google.ads.google_ads.errors import GoogleAdsException
+from google.ads.google_ads.client import GoogleAdsClient
+
+_DEFAULT_PAGE_SIZE = 1000
+ 
+ 
+def display_categories(categories,prefix=''):
```

Comment:
Add spaces in between these parameters here and in the rest of the function declarations.